leakcanary-android/src/main/java/com/squareup/leakcanary/internal/LeakCanaryInternals.java

```diff
@@ -107,6 +107,14 @@ public static void setEnabled(Context context, final Class<?> componentClass,
     });
   }
 
+  /** Based on http://stackoverflow.com/a/3758880. */
+  public static String humanReadableByteCount(long bytes) {
```

Comment:
I think you want `KiB`, `MiB`, etc. What you have is incorrect.
http://en.wikipedia.org/wiki/Binary_prefix